ketcher-autotests/tests/Structure-Creating-&-Editing/Simple-Objects/action-on-simple-objects.spec.ts

```diff
@@ -0,0 +1,242 @@
+import { Page, test, expect } from '@playwright/test';
+import {
+  LeftPanelButton,
+  openFileAndAddToCanvas,
+  waitForPageInit,
+  waitForRender,
+  drawBenzeneRing,
+  resetCurrentTool,
+  selectLeftPanelButton,
+  takeEditorScreenshot,
+  clickInTheMiddleOfTheScreen,
+  dragMouseTo,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  selectDropdownTool,
+  copyAndPaste,
+  selectTopPanelButton,
+  TopPanelButton,
+  saveToFile,
+  receiveFileComparisonData,
+  pressButton,
+  STRUCTURE_LIBRARY_BUTTON_NAME,
+} from '@utils';
+import { getKet } from '@utils/formats';
+
+const ellipseWidth = 120;
+const ellipseHeight = 100;
+
+const setupEllipse = async (page: Page) => {
+  await selectLeftPanelButton(LeftPanelButton.ShapeEllipse, page);
+  const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+  const ellipseCoordinates = { x: x + ellipseWidth, y: y + ellipseHeight };
+  await clickInTheMiddleOfTheScreen(page);
+  await dragMouseTo(ellipseCoordinates.x, ellipseCoordinates.y, page);
+  return ellipseCoordinates;
+};
+
+async function setZoomInputValue(page: Page, value: string) {
+  await page.getByTestId('zoom-input').click();
+  await page.getByTestId('top-toolbar').getByRole('textbox').fill(value);
+  // await page.getByTestId('zoom-value').fill(value);
+  await page.keyboard.press('Enter');
+}
+
+async function createSomeStructure(page: Page) {
+  const a = 97;
+  const b = 79;
+  const c = 943;
+  const d = 114;
+  const e = 844;
+  const f = 579;
+  const g = 66;
+  const h = 611;
```

Comment:
I would suggest to restruct these coordinates to meaningful pair (x, y), which is clearer and more semantic.

Reply:
Done :)